x-pack/legacy/plugins/maps/public/shared/layers/styles/vector_style.js

```diff
@@ -575,13 +576,19 @@ export class VectorStyle extends AbstractStyle {
     }
 
     const iconOrientation = this._descriptor.properties.iconOrientation;
+    const rotationOffset = getMakiSymbolRotationOffset(symbolId);
     if (iconOrientation.type === VectorStyle.STYLE_TYPE.STATIC) {
-      mbMap.setLayoutProperty(symbolLayerId, 'icon-rotate', iconOrientation.options.orientation);
+      mbMap.setLayoutProperty(
+        symbolLayerId,
+        'icon-rotate',
+        rotationOffset + iconOrientation.options.orientation);
```

Comment:
I would think we wouldn't apply the offset for static styling and just stick with the default